Assets/Packages/com.studio23.ss2.splashscreensystem/Runtime/UI/SplashScreenUIManager.cs

```diff
@@ -0,0 +1,64 @@
+using Cysharp.Threading.Tasks;
+using Studio23.SS2.SplashScreenSystem.Core;
+using Studio23.SS2.SplashScreenSystem.Data;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Studio23.SS2.SplashScreenSystem.UI
+{
+    public class SplashScreenUIManager : MonoBehaviour
+    {
+        private CanvasGroup _parentCanvasGroup;
+        private SplashScreenData _splashData;
+
+        public static SplashScreenUIManager Instance;
+        public GameObject ButtonPanel;
+        public GameObject ParentPanel;
+        public TextMeshProUGUI TitleText;
+        public GameObject ScrollRect;
+        public Transform RectParent;
+
+        public GameObject ImagePrefab;
+        public List<GameObject> ImageObjects = new List<GameObject>();
+
+        public SplashScreenBehaviour SplashScreenBehaviour;
+        public TextMeshProUGUI ScrollRectText;
+
+        //Button
+        public Button AcceptBtn;
+        public Button DeclineBtn;
+
+        private void Awake()
+        {
+            ScrollRectText = ScrollRect.GetComponentInChildren<TextMeshProUGUI>();
```

Comment:
Make another function called initialize and paste all get component line there and call that function from start